feature/amazon/src/bc5SupportDev/java/com/revenuecat/purchases/amazon/storeProductConversions.kt

```diff
@@ -0,0 +1,108 @@
+package com.revenuecat.purchases.amazon
```

Comment:
since the amazon build doesn't depend on billingclient, the feature/amazon folder shouldn't need special files per flavor...the implementation for amazon should be the same regardless of the flavor being built. or am i missing something for creating this file?

Reply:
You're right, thanks for pointing it out... not sure why I moved this